src/workflows/deploy.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+set -o nounset
+set -o pipefail
+set -o errexit
+
+GIT_BRANCH="${1:-""}"
+
+fail_if_unset () {
+  VAR_NAME="$1"
+  if [[ -z "${!VAR_NAME:-""}" ]]; then
+    printf '%s not set\n' "$VAR_NAME" >&2
+    exit 1
+  fi
+}
+gwei_to_wei () {
+  GWEI="$1"
+  printf "%s000000000" "${GWEI}"
+}
+
+fail_if_unset "ETHERSCAN_API_KEY"
+fail_if_unset "INFURA_KEY"
+fail_if_unset "PK"
+
+GAS_PRICE_GWEI_MAINNET="$( \
+  curl --silent "https://api.etherscan.io/api?module=gastracker&action=gasoracle&apikey=$ETHERSCAN_API_KEY" \
```

Comment:
Any particular reason you went with etherscan gas price instead of gnosis gas station?

```
const gasStationURL = {
  1: "https://safe-relay.gnosis.io/api/v1/gas-station/",
  4: "https://safe-relay.rinkeby.gnosis.io/api/v1/gas-station/",
}
```

Just putting this here in case you change your mind!

Reply:
I also think using the Safe gas service is a neat idea. We would just need to do a little more work to parse the resulting JSON with `jq`.